backend/src/main/java/com/redhat/cloud/notifications/routers/ValidationEndpoint.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.models.EventType;
+import io.smallrye.mutiny.Uni;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/validation")
+public class ValidationEndpoint {
+
+    @Inject
+    ApplicationResources appResources;
+
+    @GET
+    public Uni<Response> validate(@QueryParam("bundle") String bundleName, @QueryParam("application") String applicationName, @QueryParam("eventtype") String eventType) {
+        return appResources.getEventType(bundleName, applicationName, eventType)
+                .onItem()
+                .transform(this::isValid);
+    }
+
+    private Response isValid(EventType e) {
+        if (e != null) {
+            return Response.ok().entity("bundle found").build();
+        } else {
```

Comment:
This `else` block will be reached with the current tests, but not with the runtime code. When the event type is not found, `ApplicationResources#getEventType` will throw a `NoResultException`. See [EventConsumer](https://github.com/RedHatInsights/notifications-backend/blob/8ba668f4dd6a10c075eb8424d32c679b06ad6e95/backend/src/main/java/com/redhat/cloud/notifications/events/EventConsumer.java#L136).

Reply:
I think this is now unreachable code (even on tests) - We would probably could get rid of this function as well